packages/driver/cypress/integration/commands/assertions_spec.js

```diff
@@ -2492,7 +2503,7 @@ describe('src/cy/commands/assertions', () => {
 
         cy.get('button:first').should('have.focus')
         .then(() => {
-          expect(stub).to.be.calledThrice
+          expect(stub).to.be.calledTwice
```

Comment:
It should be called twice because it was checked twice in L2502 and L2504. But before this PR, it was called twice in L2504, that's why it was thrice.